lib/features/family/features/home_screen/presentation/pages/family_home_screen.dart

```diff
@@ -59,100 +59,108 @@ class _FamilyHomeScreenState extends State<FamilyHomeScreen> {
 
   @override
   Widget build(BuildContext context) {
-    return BaseStateConsumer(
-      cubit: _cubit,
-      onCustom: (context, uiModel) {
-        createOverlay(uiModel);
-        setState(() {
-          overlayVisible = true;
-        });
-        Overlay.of(context).insert(overlayEntry!);
+    return PopScope(
```

Comment:
Can't we not make use of the FunScaffold pop scope? :)

Reply:
Possible, need to add onPopInvokedWithResult method to the funscaffold then